packages/kusama/src/kusama.governance.e2e.test.ts

```diff
@@ -0,0 +1,5 @@
+import { kusama } from '@e2e-test/networks/chains'
+
+import { Network, governanceE2ETests } from '@e2e-test/shared'
+
+governanceE2ETests(Network.Kusama, kusama)
```

Comment:
instead of passing an enum here, another way is pass something like 
```
{
  name: 'Kusama',
  addressEncoding: 2,
  anyOtherConfig: 'some kusama value'
}
```

so that all the network specific config is abstracted away from the shared test

Reply:
Great idea, thank you for the suggestion.